.changeset/dry-eels-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Allows the user to change the placement of the dev toolbar.
```

Comment:
```suggestion
Adds a new dev toolbar settings option to change the horizontal placement of the dev toolbar on your screen: bottom left, bottom centre, or bottom right.
```
What about something like this? I like indicating HOW people can access this (it's in the Settings menu), and setting up the expectation that they can't put it ANYWHERE (drag it around, top of the screen etc.)

Reply:
That looks great! Thanks for your suggestion https://github.com/withastro/astro/pull/10591/commits/ee1db274539afbb51eb335a36655b74ec5ca6747